en/asgardeo/docs/apis/organization-apis/restapis/scim-bulk.yaml

```diff
@@ -0,0 +1,192 @@
+openapi: 3.0.1
+info:
+  title: SCIM 2.0 Bulk API
+  description: |
+    This document specifies **Organization level SCIM 2.0 Bulk Resource Management RESTful API** for Asgardeo.
+  version: 1.0.0
+servers:
+  - url: https://api.asgardeo.io/t/{organization-name}/o/scim2
+
+security:
+  - OAuth2: []
+
+paths:
+  /Bulk:
+    post:
+      tags:
+        - Bulk Endpoint
+      summary: Create, Update, Delete Users and Groups in bulk and update Roles in bulk
+      description: |
+        This API is used to create/update/replace/delete SCIM resources in bulk. You need to specify different operations in the request payload corresponding to the type of bulk update: </br>
+          - **POST** operation: Adds new resources.
+          - **PUT** operation: Replaces an existing resource.
+          - **PATCH** operation: Updates attributes of an existing resource.
+          - **DELETE** operation: Removes an existing resource.
+        Learn more about [defining batch operations](../../scim2-batch-operations).
+
+         <b>Scope(Permission) required:</b> `internal_org_bulk_resource_create`
+      operationId: createBulkUsers
+      requestBody:
+        content:
+          application/scim+json:
+            schema:
+              $ref: '#/components/schemas/BulkUserRequestObject'
+        required: false
+      responses:
+        201:
+          description: Valid user is created
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/BulkUserResponseObject'
+        401:
+          description: Unauthorized
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/ErrorUnauthorized'
+        403:
+          description: Forbidden
+          content:
+            application/scim+json:
+              schema:
+                $ref: '#/components/schemas/ErrorForbidden'
+        409:
+          description: Users already exists
+          content: {}
+      x-codeSamples:
+        - lang: Curl
+          source: |
+            curl -X 'POST' \
+            'https://api.asgardeo.io/t/{organization-name}/o/scim2/Bulk' \
+            -H 'accept: application/scim+json' \
+            -H 'Content-Type: application/scim+json' \
+            -H 'Authorization: Bearer {bearer_token}'
+            -d '{
+            "failOnErrors": 1,
+            "schemas": [
+              "urn:ietf:params:scim:api:messages:2.0:BulkRequest"
+            ],
+            "Operations": [
+              {
+                "method": "POST",
+                "path": "/Users",
+                "bulkId": "ytrewq",
+                "data": {
+                  "schemas": [
+                    "urn:ietf:params:scim:schemas:core:2.0:User"
+                  ],
+                  "username": "jesse",
```

Comment:
I guess the userstore domain need to be appended here, otherwise it will be trated as a primary userstore user

Reply:
https://github.com/wso2/docs-is/pull/4563/commits/ee322b43c5672d4c62bb028ce836acdc9b21a53e